x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -8,14 +8,18 @@ import type { SavedObjectsClientContract } from '@kbn/core/server';
 import { transformError } from '@kbn/securitysolution-es-utils';
 import type { AgentPolicy, PackagePolicy } from '@kbn/fleet-plugin/common';
 import { CspRuleTemplate } from '../../../common/schemas';
-import { CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE } from '../../../common/constants';
+import {
+  CNVM_POLICY_TEMPLATE,
```

Comment:
not related to this PR but pay attention this is misleading
we do not hold any policy_template named CNVM